src/test/url-handling.test.js

```diff
@@ -187,13 +187,13 @@ describe('url handling tracks', function() {
     });
 
     it('will not accept invalid tracks in the thread order', function() {
-      const { getState } = initWithSearchParams('?globalTrackOrder=1-0');
+      const { getState } = initWithSearchParams('?globalTrackOrder=10');
       expect(urlStateSelectors.getGlobalTrackOrder(getState())).toEqual([1, 0]);
     });
 
     it('will not accept invalid hidden threads', function() {
       const { getState } = initWithSearchParams(
-        '?hiddenGlobalTracks=0-8-2-a&thread=1'
+        '?hiddenGlobalTracks=082~&thread=1'
```

Comment:
Hmm, actually, I should change this. The old test used `a` as something that's unparseable and creates a NaN (I think). With the new implementation, we parse everything, and unknown characters just create incorrect numbers with some zeros (because of the default branch in `bitsFromEncodingDigit`). So then this test input is only invalid because it refers to track indexes that are larger than the total track count. I'll just use `08` or something like that, it achieves the same.